geth/rpc/client_test.go

```diff
@@ -135,7 +135,7 @@ func (s *RPCTestSuite) TestCallRPC() {
 				"gas": "0x76c0",
 				"gasPrice": "0x9184e72a000",
 				"value": "0x9184e72a",
-				"data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"}],"id":1}`,
+				"data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"}]}`,
```

Comment:
Why?

Reply:
Refactoring artifact :/ (ids were removed, then added with workaround and returned to tests). Will add this one.

Reply:
There is no actual test in this case though.

Reply:
Sorry?

Reply:
@tiabc original code: https://github.com/status-im/status-go/blob/develop/geth/node/rpc_test.go#L138

This function is just copy paste from `node/rpc_test.go`, the logic shouldn't change and even if the whole test is flawed I don't want to fix it as a part of this PR.

Reply:
But it's not copy-paste: in your case `id` has been removed.

Reply:
At some point decision was to have no `ids` in response for `CallRaw()`, so tests had to be cleaned up, and this one was "cleaned up" as well. As it doesn't perform actual test, tests were passing. Anyway, returned `id` back.